crates/re_viewer/src/app.rs

```diff
@@ -95,7 +94,7 @@ pub struct App {
     rx: Receiver<LogMsg>,
 
     /// Where the logs are stored.
-    log_dbs: HashMap<RecordingId, LogDb>,
+    log_db_hub: crate::LogDbHub,
```

Comment:
Again, not really a fan of keeping `log` here since it is not one of store, recording, or blueprint.